packages/app/src/cli/services/function/binaries.test.ts

```diff
@@ -215,7 +217,9 @@ describe('functionRunner', () => {
     })
   })
 
-  test('function-runner installs and runs', async () => {
+  // This shouldn't take 20 seconds to run but sometimes the network hangs and
+  // retries need to happen.
+  test('function-runner installs and runs', {timeout: 20000}, async () => {
```

Comment:
This seems like more of an integration test. I wonder if we should mock the response so that we're not dependent on an external request. @shauns Do you have opinions on this?